CPS-XXXX/README.md

```diff
@@ -0,0 +1,82 @@
+---
+CPS: ?
+Title: Ouroboros not accounting for typical global network delays causes centralisation
```

Comment:
```suggestion
Title: Block Delay Centralisation
```
The originally presented title is a "motivation" rather than a title & we need to be more concise here to blend in with other proposals & problem statements... it's not just about fitting into smaller space in a table (though it helps).

The title you put into your branch, which became the initial title for this PR, is "just right" in my opinion: neither too general nor too specific.  I would have been tempted to say `Geographical` centralisation if not for the fact that the same problem (and same solution) you are talking about can also be caused by variations in network media rather than asymmetry in network topology.